lib/components/artist/shows.js

```diff
@@ -0,0 +1,44 @@
+/* @flow */
+'use strict';
+
+import React from 'react-native';
+const { View, Text, StyleSheet, Image } = React;
+
+import Headline from '../text/headline';
+import SerifText from '../text/serif'
+
+export default class Shows extends React.Component {
+
+	render() {
+	  return (
+	  	<View style={styles.container}>
+		  	<SerifText style={styles.title}> Current & Upcoming Shows </SerifText>
+		  	<Text style={styles.sansSerifText}> {this.props.show.location} </Text>
+		  	<Text style={styles.sansSerifText}> {this.props.show.partner_name} </Text>
```

Comment:
If I try to make this into a Headline, I get the following error: ```this.props.children.toUppercase is not a function``` and I see that that's what the Headline component is calling. Why would this work elsewhere but not here?